.gitignore

```diff
@@ -108,7 +108,3 @@ frontend/test_results
 frontend/cypress/snapshots/darwin
 frontend/cypress/snapshots/linux/1x
 
-########################################################################
-# VSCode
-########################################################################
-.vscode/
```

Comment:
**note:** This is an intentional change in order to check this file in. Note that this may inadvertently cause those with existing `.vscode` configurations to run into a merge conflict upon pulling this change once it's checked in.

Reply:
Maybe it would be good to still exclude the rest of the folder:

```
.vscode/*
!.vscode/launch.json
```

Since the rest of files there do not seem as shareable?

Reply:
Great suggestion, will update!